app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -758,8 +743,10 @@ public void onResume() {
 
         // Also apply colours to UI elements
         int darkenedColor = ColorUtils.blendARGB(backgroundHeaderColor, Color.BLACK, 0.1f);
-        barcodeScaler.setProgressTintList(ColorStateList.valueOf(darkenedColor));
-        barcodeScaler.setThumbTintList(ColorStateList.valueOf(darkenedColor));
+        barcodeHeightScaler.setProgressTintList(ColorStateList.valueOf(darkenedColor));
+        barcodeHeightScaler.setThumbTintList(ColorStateList.valueOf(darkenedColor));
+        barcodeWidthScaler.setProgressTintList(ColorStateList.valueOf(darkenedColor));
+        barcodeWidthScaler.setThumbTintList(ColorStateList.valueOf(darkenedColor));
```

Comment:
And now it's suddenly

Height
Height
Width
Width

These inconsistencies make things really hard to maintain and prone to future bugs.